docs/source/main_classes/trainer.rst

```diff
@@ -570,22 +686,28 @@ cell with:
     EOT
 
 
-That's said if the script is not in the notebook cells, you can launch ``deepspeed`` normally via shell from a cell
-with:
+If the training script is in a normal file and not in the notebook cells, you can launch ``deepspeed`` normally via
+shell from a cell. For example, to use ``run_translation.py`` you would launch it with:
 
 .. code-block::
 
-   !deepspeed examples/seq2seq/run_translation.py ...
+   !cd transformers; deepspeed examples/seq2seq/run_translation.py ...
```

Comment:
Maybe add the clone command? Users won't have the repo in the current folder otherwise.